nly

```diff
@@ -0,0 +1,4 @@
+Inspecting 1 file
+.
+
+0 files inspected, no offenses detected
```

Comment:
Should this file be ignored?

Reply:
looks like output from rubocop or other linter/tool. will need to be removed I think

Reply:
I have deleted this file now. Thanks for pointing it out.